lua/nvim-treesitter/parsers.lua

```diff
@@ -1094,6 +1094,23 @@ list.nickel = {
   },
 }
 
+list.nim = {
+  install_info = {
+    url = "https://github.com/alaviss/tree-sitter-nim",
+    files = { "src/parser.c", "src/scanner.cc" },
+    branch = "main",
```

Comment:
```suggestion
```
Not needed if it's the default branch; you do need to add a lockfile entry for the parsers, though.